.github/workflows/docker-singularity-publish.yml

```diff
@@ -14,25 +14,28 @@ on:
     branches: [ main ]
     # Publish semver tags as releases.
     tags: [ 'v*.*.*' ]
+  pull_request:
```

Comment:
as I understand this will be removed before merging?

Reply:
Correct!

Reply:
Should we remove now?

Reply:
```suggestion
```